algorithms/linfa-clustering/src/gaussian_mixture/algorithm.rs

```diff
@@ -596,7 +596,7 @@ mod tests {
         assert!(
             match gmm.expect_err("should generate an error with reg_covar being nul") {
                 GmmError::LinalgError(e) => match e {
-                    LinalgError::Lapack { return_code: 1 } => true,
+                    LinalgError::Lapack(_) => true,
```

Comment:
you have to check of the `return_code` here

Reply:
You're right, I took a shortcut, Lapack error has changed, I will check

Reply:
done

Reply:
you can also click on `Resolve conversation`

Reply:
Yeah, I always tend to feel that the conversation should be resolved by the reviewer.